modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -129,6 +135,53 @@ public void testConfigurationArrayValues() throws Exception {
 			(String[])properties.get("configUntypedStringArray"));
 	}
 
+	@Test
+	public void testConfigurationDeprecatedFileExtension() throws Exception {
+		String configurationPidDeprecated = _CONFIGURATION_PID_PREFIX.concat(
+			".testConfigurationDeprecatedFileExtension");
+		String configurationPid = _CONFIGURATION_PID_PREFIX.concat(
+			".testDummy");
+
+		String content = "testKey=\"testValue\"";
+
+		Path configPathDeprecated = Paths.get(
+			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR,
+			configurationPidDeprecated.concat(".cfg"));
+
+		_configurationPath = Paths.get(
+			PropsValues.MODULE_FRAMEWORK_CONFIGS_DIR,
+			configurationPid.concat(".config"));
+
+		try (CaptureAppender captureAppender =
+				Log4JLoggerTestUtil.configureLog4JLogger(
+					"com.liferay.portal.file.install.internal.configuration." +
+						"ConfigurationFileInstaller",
+					Level.WARN)) {
+
+			Files.write(
+				configPathDeprecated,
+				content.getBytes(Charset.defaultCharset()));
+
+			_configuration = _createConfiguration(configurationPid, content);
```

Comment:
It doesn't look like you're doing anything with this. You need to give it a .cfg file with properties inside and assert that those properties aren't being read by the system